test/malloc_compliance_tests.cpp

```diff
@@ -85,12 +85,11 @@ void calloc_compliance_test(umf_memory_pool_handle_t hPool) {
     // Checking that the memory returned by calloc is zero filled
     for (int i = 0; i < ITERATIONS; i++) {
         alloc_size = rand_alloc_size(MAX_ALLOC_SIZE);
-        alloc_ptr[i] = umfPoolCalloc(hPool, i + 1, alloc_size);
+        alloc_ptr[i] = umfPoolCalloc(hPool, 1, alloc_size);
```

Comment:
Could you make it 2 blocks of memory instead of 1? This should still be much less memory allocated than in the previous implementation but still using calloc's feature of allocating multiple blocks of memory.

Reply:
Done